agents/holo_fpga/roach_agent.py

```diff
@@ -0,0 +1,182 @@
+import argparse
+import os
+import numpy as np
+import txaio
+import yaml
+import time
+
+ON_RTD = os.environ.get("READTHEDOCS") == "True"
+if not ON_RTD:
+    import casperfpga
+    from holog_daq import fpga_daq3, synth3
+    from ocs import ocs_agent, site_config
+    from ocs.ocs_twisted import TimeoutLock
+
+
+class FPGAAgent:
+    """
+    Agent for connecting to the Synths for holography
+
+    Args: $OCS_CONFIG_DIR holog_config.yaml
+    """
```

Comment:
Similar to in the `SynthAgent` this needs a bit of reformatting to:
```
"""
Agent for connection to the Synths for holography.

Args:
    config_file (str): Path to holography config file
"""
```

Or similar. It's just that right now this produces:
![image](https://user-images.githubusercontent.com/7691438/182630530-a7c75b84-06c6-40a8-9aba-f452d44a681d.png)

when it should produce a nicely formatted "Parameters" section, with a bulleted list like this example from the 372 Agent:
![image](https://user-images.githubusercontent.com/7691438/182630675-a9371e4e-1fbf-411f-9114-e66085488688.png)